tests/jsanalyze_test.js

```diff
@@ -88,9 +103,10 @@ describe('jsanalyze', function () {
 		it('generates source maps inline into generated js file and removes sourcesContent for android platform', function () {
 			const inputJSFile = path.join(__dirname, 'resources/input.js');
 			const contents = fs.readFileSync(inputJSFile, 'utf-8');
-			const expectedSourceMap = fs.readJSONSync(inputJSFile + '.map');
+			var expectedSourceMap = fs.readJSONSync(inputJSFile + '.map');
```

Comment:
Use `let` instead of `var`. There's a couple occurrences below.

Reply:
:+1: fixed